README.md

```diff
@@ -2175,3 +2211,5 @@ https://github.com/abhinavsingh/proxy.py/issues/new
 - Single threaded server.
 
 For detailed changelog refer to release PRs or commit history.
+ history.
+ history.
```

Comment:
```suggestion
```

Reply:
Thank you.  I think `check.py` is faulty and inserting this automagically.  Lemme address it.